test/test_payment_initiation.rb

```diff
@@ -9,10 +9,14 @@ def test_all_routes
     create_recipient_response = client.payment_initiation.create_recipient(
       'John Doe',
       'GB33BUKB20201555555555',
-      street:      ['Street Name 999'],
-      city:        'City',
-      postal_code: '99999',
-      country:     'GB'
+      {
+        street:      ['Street Name 999'],
+        city:        'City',
+        postal_code: '99999',
+        country:     'GB'
+      },
+      account: '5555555',
```

Comment:
nit: wondering if `account` and `sort_code` should also be nested since they're part of the same hash.

Reply:
the linter complains if we do not do it like this.